tools/type_whisperer/BUILD

```diff
@@ -35,7 +35,38 @@ py_binary(
     ],
 )
 
+# Pack API type database text file into a char* string that can be referenced
+# at the C++ level.
+genrule(
+    name = "api_type_db_genrule",
+    srcs = ["api_type_db.generated.pb_text"],
```

Comment:
This one doesn't exist, define an `type_database` target rule like https://github.com/envoyproxy/envoy/blob/master/tools/api_proto_plugin/BUILD#L36-L40 in this file or just rely on default_type_db?